app/views/home/index.html.erb

```diff
@@ -48,11 +48,25 @@
     <%= render 'inbox_module', inbox_comments: @homepage.inbox_comments %>
   <% end %>
 
-  <% unless Rails.env.test? %>
-    <div class="latest tweets module">
-      <h3 class="heading"><%= ts('Tweets') %></h3>
-      <a class="twitter-timeline" href="https://twitter.com/otw_status/lists/otw-tweets" data-dnt="true" data-height="600"><%= ts('Tweets from https://twitter.com/otw_status/lists/otw-tweets') %></a>
-      <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
-    </div>
-  <% end %>
+  <div class="social module">
+    <h3 class="heading"><%= t(".social.heading") %></h3>
+    <p class="note"><%= t(".social.note_html", other_outlets_link: link_to(t(".social.other_outlets"), "https://www.transformativeworks.org/where-find-us/")) %></p>
+
+    <ul>
+      <li class="twitter">
+        <a class="resp-sharing-button__link" href="https://twitter.com/AO3_Status" rel="noopener" aria-label="<%= t(".social.twitter") %>">
+          <div class="resp-sharing-button resp-sharing-button--twitter resp-sharing-button--large"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solid">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M23.44 4.83c-.8.37-1.5.38-2.22.02.93-.56.98-.96 1.32-2.02-.88.52-1.86.9-2.9 1.1-.82-.88-2-1.43-3.3-1.43-2.5 0-4.55 2.04-4.55 4.54 0 .36.03.7.1 1.04-3.77-.2-7.12-2-9.36-4.75-.4.67-.6 1.45-.6 2.3 0 1.56.8 2.95 2 3.77-.74-.03-1.44-.23-2.05-.57v.06c0 2.2 1.56 4.03 3.64 4.44-.67.2-1.37.2-2.06.08.58 1.8 2.26 3.12 4.25 3.16C5.78 18.1 3.37 18.74 1 18.46c2 1.3 4.4 2.04 6.97 2.04 8.35 0 12.92-6.92 12.92-12.93 0-.2 0-.4-.02-.6.9-.63 1.96-1.22 2.56-2.14z" /></svg>
+          </div><%= t(".social.twitter") %></div>
+        </a>
+      </li>
+      <li class="tumblr">
+        <a class="resp-sharing-button__link" href="https://ao3org.tumblr.com" aria-label="<%= t(".social.tumblr") %>">
+          <div class="resp-sharing-button resp-sharing-button--tumblr resp-sharing-button--large"><div aria-hidden="true" class="resp-sharing-button__icon resp-sharing-button__icon--solid">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.5.5v5h5v4h-5V15c0 5 3.5 4.4 6 2.8v4.4c-6.7 3.2-12 0-12-4.2V9.5h-3V6.7c1-.3 2.2-.7 3-1.3.5-.5 1-1.2 1.4-2 .3-.7.6-1.7.7-3h3.8z" /></svg>
+            </div><%= t(".social.tumblr") %></div>
+        </a>
+      </li>
+    </ul>
+  </div>
```

Comment:
Might be worth creating a `_twitter_button.html.erb` and a `_tumbler_button.html.erb` partials rather than copy-pasting? Especially as the buttons in `_share.html.erb`already look like partials in waiting, with params and all (twitter_text, twitter_url, etc.)